cloudflare/workers-sdk
# C3-name-validation-fix

If you hit enter when asked for the name of the project, you expect it
to use the default value. But the project name validation was then failing
as it was receiving undefined for the value of the input rather than the
default value.

Now the validator will be passed the default if no value was provided.

**Author has included the following, where applicable:**

- [N/A] Tests - it is not really possible to automate a test for this as it requires interaction.
- [x] Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))

**To test manually:**

1. Run `npm run build` in the root to ensure the binaries are up to date.
2. Then run `node ./packages/create-cloudflare/dist/cli.js`
3. And hit enter.

**Reviewer is to perform the following, as applicable:**

- Checked for inclusion of relevant tests
- Checked for inclusion of a relevant changeset
- Checked for creation of associated docs updates
- Manually pulled down the changes and spot-tested
